Code/GraphMol/MolDraw2D/MolDraw2DHelpers.h

```diff
@@ -57,6 +57,7 @@ typedef std::map<int, DrawColour> ColourPalette;
 typedef std::vector<double> DashPattern;
 
 //! use the RDKit's default palette r
+// 201 is for hydrogens when the H is not being drawn.
```

Comment:
suggestion for here and the other instances of this comment:
```suggestion
// 201 is for hydrogens when atom symbols are not being drawn.
```